frontend/src/modules/frameworkImport/views/frameworkImportView.js

```diff
@@ -65,12 +67,112 @@ define(function(require){
         },
 
         error: _.bind(this.onAjaxError, this),
-        success: _.bind(this.onFormSubmitSuccess, this)
+        success: _.bind(this.displayDetails, this)
```

Comment:
Should be going for the native `.bind()`s really. Up to you whether you can be bothered to switch them!

Reply:
See also the new code on lines 230 & 231.